test/smoketest/main.py

```diff
@@ -186,35 +296,65 @@ def publish_ballots_to_audit(seed, cvrs):
             if i in new_list:
                 cvr['record_type'] = 'AUDITOR_ENTERED'
                 selected.append(cvr)
-                print("Selected cvr %d: id: %d RecordID: %s" % (i, cvr['id'], cvr['imprinted_id']))
+                logging.info("Selected cvr %d: id: %d RecordID: %s" % (i, cvr['id'], cvr['imprinted_id']))
 
         ballots_to_audit.append([county_id, selected])
 
     return ballots_to_audit
 
 
-def upload_files(baseurl, s):
+def upload_files(ac, s):
     """Directly upload files, which zerotest doesn't support.
     See "File upload via POST request not working: Issue #12"
      https://github.com/jjyr/zerotest/issues/12
     """
 
-    upload_manifest(baseurl, s, "../e-1/arapahoe-manifest.csv",
-                "42d409d3394243046cf92e3ce569b7078cba0815d626602d15d0da3e5e844a94")
+    # TODO: predefine more manifests
+    manifestfile = "../e-1/arapahoe-manifest.csv"
+    hash = "42d409d3394243046cf92e3ce569b7078cba0815d626602d15d0da3e5e844a94"
 
-    v = 1
-    if v == 1:
-        upload_cvrs(baseurl, s, "../e-1/arapahoe-regent-3-clear-CVR_Export.csv",
-                    "413befb5bc3e577e637bd789a92da425d0309310c51cfe796e57c01a1987f4bf")
-    if v == 2:
-        upload_cvrs(baseurl, s, "../dominion-2017-CVR_Export_20170310104116.csv",
-                    "4e3844b0dabfcea64a499d65bc1cdc00d139cd5cdcaf502f20dd2beaa3d518d2")
+    manifest = ac.args.manifest # TODO finish when we have more manifests and they make a difference
 
-    if v == 3:
-        upload_cvrs(baseurl, s, "../Denver2016Test/CVR_Export_20170804111144.csv",
-                    "1def4aa4c0e1421b4e5adcd4cc18a8d275f709bc07820a37e76e11a038195d02")
+    if ac.args.ye_olde_upload:
```

Comment:
I would remove this argument and logic. The old upload endpoints will go away soon, and there isn't much point in doing further testing on them; moreover, if somebody for some reason wants to test an old version, they can use the smoketest script that's current at that version in the repo.

Reply:
For now I still want to be able to test old branches, with the flexibility of this code, e.g. for timing, and hate to be without the code that is necessary.  I'll remove them later. And I like the name, go figure.